lib/rules/no-mut-helper.js

```diff
@@ -0,0 +1,70 @@
+'use strict';
+
+const createErrorMessage = require('../helpers/create-error-message');
+const Rule = require('./base');
+
+const message =
+  'Unexpected usage of mut helper. If using mut as a setter, consider using a JS action instead.';
+function generateMessageWithAlternative(setAlternative) {
+  return `Unexpected usage of mut helper. If using mut as a setter, consider using a JS action or ${setAlternative} instead.`;
+}
+
+module.exports = class NoMutHelper extends Rule {
+  parseConfig(config) {
+    if (!config || !config.setterAlternative) {
+      return {};
+    } else if (typeof config.setterAlternative !== 'string') {
+      let errorMessage = createErrorMessage(
+        this.ruleName,
+        [
+          '  * object -- An object with the following key:',
+          '    * `setterAlternative` -- String name of a helper that could replace mut as a setter',
+        ],
+        config
+      );
+
+      throw new Error(errorMessage);
+    } else {
+      return config;
+    }
+  }
+
+  visitor() {
+    return {
+      MustacheStatement(node) {
+        this._checkForMutHelper(node);
+      },
+
+      BlockStatement(node) {
```

Comment:
loks like `mut` could not be path in block statement

```hbs
{{#mut ... }}

{{/mut}}}
```

Reply:
ah, good point Alex. this was accidental copy paste residue from the similar `no-unbound` rule.

fixed and thanks for the suggestion.